resources/puppetlabs/lein-ezbake/template/foss/ext/debian/ezbake.init.erb

```diff
@@ -118,27 +109,24 @@ get_status_q()
 do_restart()
 {
     do_stop
-    case "$?" in
-        0|1)
```

Comment:
so the `--oknodo` flag makes me doubt we would ever get a non-zero return on the stop command.